crates/rspack_plugin_css/src/parser_and_generator/mod.rs

```diff
@@ -164,7 +164,7 @@ impl ParserAndGenerator for CssParserAndGenerator {
           range,
           kind,
         } => {
-          if request.is_empty() {
+          if request.trim().is_empty() {
             continue;
```

Comment:
Should we emit a warning/error here? Does webpack experiments.css emit a warning/error? cc @GiveMe-A-Name

Reply:
webpack do not emit any warning, [webpack remove the unnecessary spaces in url(" xxx ")](https://github.com/webpack/webpack/blob/main/lib/css/CssParser.js#L75-L78).
I think we no need to emit warning.